### ISSUE TITLE
Add ability to send a request from a script

### ISSUE THREAD

**Issue by HFPasfho** [Waboodoo/HTTP-Shortcuts#422]
### Describe the solution you'd like
It would be great if you can send a request (fetch/XHR) in script. Currently, to react and send a request based on the received response, you have to create a separate shortcut and call it from within the shortcut and additionally use global variables to pass content to this request. This multiplies the number of shortcuts and global variables and unnecessarily complicates the implementation of very simple concepts.

**Comment by Waboodoo** [comment]
I agree that this could and should be simpler. However, I currently don't know how I could properly implement this, without complicating things. I'm also hesitating a bit because this would add more redundant features and with that a potential for confusion.

**Comment by Waboodoo** [comment]
As of version 3.21.0, there is a newly added function `sendHttpRequest()` (documentation [here](https://http-shortcuts.rmy.ch/scripting#send-http-request)), which allows to make same basic HTTP requests directly. I might extend it in the future, but for now it is intentionally kept to a minimum in what it supports.
